rma_repair/models/rma_order_line.py

```diff
@@ -142,6 +142,10 @@ def _onchange_operation_id(self):
     def _compute_qty_to_deliver(self):
         res = super(RmaOrderLine, self)._compute_qty_to_deliver()
         for rec in self.filtered(lambda l: l.delivery_policy == 'repair'):
-            rec.qty_to_deliver = rec.qty_repaired - rec.qty_delivered - \
-                rec.qty_to_pay
+            rec.qty_to_deliver = rec.qty_repaired - rec.qty_delivered
+        return res
+
+    def rma_repair_make_invoice(self):
+        res = self.env['repair.order.make_invoice'].with_context(
```

Comment:
If there is more than one repair in the RMA, IMO you should group the invoices in one using the `group` field in the `'repair.order.make_invoice'` model.

Reply:
done :)